src/gobprepare/sql.py

```diff
@@ -0,0 +1,96 @@
+# SQL constants
+SQL_TYPE_CONVERSIONS = {
+    "String": "character varying",
+    "DateTime": "timestamp without time zone",
+    "Int": "integer",
+    "Boolean": "boolean",
+    "GeoJSON": "geometry",
+}
+SQL_QUOTATION_MARK = "'"
+
+
+def _quote(name):
+    """
+    Quote all SQL identifiers (schema, table, column names)
+    to prevent weird errors with SQL keywords accidentally being used in identifiers.
+
+    Note that quotation marks may differ per database type.
+    Current escape char works for PostgreSQL
+
+    :param name:
+    :return:
+    """
+    QUOTE_CHAR = '"'
+    return f"{QUOTE_CHAR}{name}{QUOTE_CHAR}"
+
+
+def _create_field(name, type, description):
+    """
+    Create a database field
+
+    :param name:
+    :param type:
+    :param description:
+    :return: dict containing database field properties
+    """
+    return {
+        'name': _quote(name),
+        'type': SQL_TYPE_CONVERSIONS[type],
+        'description': description
+    }
+
+
+def get_create_schema_sql(name):
+    """
+    Returns a SQL statement to create a schema with the given name
+
+    :param name:
+    :return:
+    """
+    return f"""
+-- DROP SCHEMA {name} CASCADE;
+CREATE SCHEMA IF NOT EXISTS {_quote(name)};
+COMMIT;
+"""
+
+
+def get_create_table_sql(schema, table_name, description, meta_fields, field_names):
```

Comment:
Similar code exists in the Prepare cloner. Combine both? Same goes for the quote function above. Very similar code in the same repo.